package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha tests/index.js --exit --require babel-core/register"
```

Comment:
This script doesn't look like it will work as intended. Check out this [link](https://babeljs.io/setup#installation) on how to setup `mocha` and `babel`. Also, please a simple test so that we can assert that the setup runs fine.